tests/Place/Canonical/DBALDuplicatePlaceRepositoryTest.php

```diff
@@ -173,4 +173,52 @@ public function it_can_get_the_duplicates_of_a_place(): void
             $this->duplicatePlaceRepository->getDuplicatesOfPlace('64901efc-6bd7-4e9d-8916-fcdeb5b1c8ad')
         );
     }
+
+    public function test_places_no_longer_in_cluster(): void
+    {
+        $this->getConnection()->insert(
+            'duplicate_places_import',
+            [
+                'cluster_id' => 'cluster_2',
+                'place_uuid' => '19ce6565-76be-425d-94d6-894f84dd2947',
+            ]
+        );
+
+        $this->assertEquals(
+            [
+                '1accbcfb-3b22-4762-bc13-be0f67fd3116',
+                '4a355db3-c3f9-4acc-8093-61b333a3aefb',
+                '526605d3-7cc4-4607-97a4-065896253f42',
+                '64901efc-6bd7-4e9d-8916-fcdeb5b1c8ad',
+            ],
+            $this->duplicatePlaceRepository->getPlacesNoLongerInCluster()
+        );
+    }
+
+    public function test_clusters_to_be_removed(): void
+    {
+        $this->getConnection()->insert(
+            'duplicate_places_import',
+            [
+                'cluster_id' => 'cluster_2',
+                'place_uuid' => '19ce6565-76be-425d-94d6-894f84dd2947',
+            ]
+        );
+
+        $this->assertEquals(
+            [
+                'cluster_1',
+            ],
+            $this->duplicatePlaceRepository->getClustersToBeRemoved()
+        );
+    }
+
+
+    public function test_delete_cluster(): void
+    {
+        $this->duplicatePlaceRepository->deleteCluster('cluster_1');
+
+        $raw = $this->connection->fetchAssociative('select count(*) as total from duplicate_places where cluster_id = :cluster_id', ['cluster_id' => 'cluster_1']);
```

Comment:
I would put in the extra effort to also use the query builder in tests, the query builder fluent syntax is so easy to read.